docs/components/movement-sensor/merged.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Configure a merged movement sensor"
+linkTitle: "merged"
+weight: 40
+type: "docs"
+description: "Configure a merged movement sensor."
+images: ["/icons/components/imu.svg"]
+# SMEs: Rand
+---
+
+The `merged` movement sensor model is an abstraction that combines data from multiple movement sensors.
+
+This allows you to aggregate the API methods supported by multiple sensors into a singular sensor client, effectively merging the models of the individual resources.
+
+In other words, the models of the `merged` sensors are combined, making the movement sensor API methods each aggregated model supports available on the `merged` model.
+
+This is especially useful if you want to get readings of position and orientation *or* linear and angular velocity at the same time, which are normally separately supported and returned by [`GPS`](/components/movement-sensor/#configuration) or [`IMU`](/components/movement-sensor/#configuration) models, respectively.
+
+{{% alert title="Usage Tip: Navigation Service" color="tip" %}}
+Aggregating `Position()` from a [`GPS`](/components/movement-sensor/#configuration) and `Orientation()` from an [`IMU`](/components/movement-sensor/#configuration) movement sensor in a `merged` model allows you to reduce velocity error when your robot is trying to [navigate](/services/navigation/).
```

Comment:
```suggestion
To reduce velocity error when your robot is using the [navigation service](/services/navigation/), aggregate `Position()` from a [`GPS`](/components/movement-sensor/#configuration) and `Orientation()` from an [`IMU`](/components/movement-sensor/#configuration) movement sensor in a `merged` model.
```
[opt] Generally I like saying "why" before "how," and this rewrite makes the end of this sentence less redundant with the following sentence, but I can see the argument for leaving this as-is so up to you.